src/renderers/WebGLRenderer.js

```diff
@@ -1889,9 +1894,11 @@ function WebGLRenderer( parameters ) {
 
 				}
 
+				const halfFloatSupportedByExt = textureType === HalfFloatType && ( extensions.get( 'EXT_color_buffer_half_float' ) || ( capabilities.isWebGL2 && extensions.get( 'EXT_color_buffer_float' ) ) );
```

Comment:
NIT: Instead of using `WebGLExtension.get()` in `readRenderTargetPixels()`, it should be sufficient to use `has()`.